dronekit/__init__.py

```diff
@@ -254,6 +252,148 @@ def __init__(self, distance, voltage):
     def __str__(self):
         return "Rangefinder: distance={}, voltage={}".format(self.distance, self.voltage)
 
+class Version(object):
+    """
+    Version numbers.
+
+    An object of this type is returned by :py:attr:`Vehicle.version`
+    The version number can be read in a few different formats. To get it in a human-readable
+    format, just print `vehicle.version`.  This might print somthing like "APM:Copter-3.3.2-rc4".
+
+    .. py:attribute:: major
+
+    Major version number (integer).
```

Comment:
Indent 4 spaces (this is how sphinx works out that it is processing a new block). In fact for any new directive you have to have a new line before the directive, a line after the directive, and then the content of the directive block must be indented by 4 spaces. So for example a note would look like this:

```
some text.

.. note::

    The note content.
    This is more note content.
```